src/app/pages/careers/prysmatic-careers.page.component.html

```diff
@@ -0,0 +1,206 @@
+                    </li>
+                    <li>
+                        Learned about our design and code review process, participate in itemizing your tasks into smaller ones based on tracking issues on Github, and contribute to code health across our project
+                    </li>
+                    <li>
+                        Worked with the full team on understanding our cloud architecture using Kubernetes, learning about instrumentation, debugging distributed systems, and production release
+                    </li>
+                    <li>
+                        Participated with other teammates in being the first line of defense against network failures, outages, and unexpected scenarios using instrumentation and best practices
+                    </li>
+                </ul>
+                <p class="text-2xl">
+                    In six months, you will have:
+                </p>
+                <ul class="bullet py-4 mb-6">
+                    <li>
+                        Have taken ownership of several features from end to end on our roadmap through maintainable code with good tests, documentation, and clear design principles
+                    </li>
+                    <li>
+                        Integrated and write your own instrumentation pipelines for analyzing the Ethereum 2.0 network through our cloud infrastructure
+                    </li>
+                    <li>
+                        Engaged with the core researchers behind the Ethereum 2.0 protocol on biweekly video calls and design discussionss
+                    </li>
+                    <li>
+                        Optimized mission-critical distributed and concurrent systems
+                    </li>
+                    <li>
+                        A deeper understanding of blockchain technology, cryptography, p2p networking, and systems architecture
+                    </li>
+                </ul>
+                <p class="text-2xl">
+                    In one year, you will have:
+                </p>
+                <ul class="bullet py-4 mb-10">
+                    <li>
+                        Built up your engineering skills by attending presentations, workshops, and other speaking engagements on behalf of Prysmatic Labs
+                    </li>
+                    <li>
+                        Shipped the real, live 2.0 Ethereum blockchain with proof of stake and sharding to production, marking one of the most exciting milestones in modern Computer Science
+                    </li>
+                    <li>
+                        Mentored future team-members and junior developers, helping them level up technically
+                    </li>
+                    <li>
+                        Identified and prevented large amounts of technical debt in our repositories through solid design principles and large-scale optimization
```

Comment:
I would hope there aren't large amounts of technical debt! Can we rephrase this to something about code health?

Reply:
Good point lol it sounds like it gives the opposite message!